Xamarin.Forms.GoogleMaps/Xamarin.Forms.GoogleMaps.UWP/PushPin.cs

```diff
@@ -66,5 +136,21 @@ void UpdateLocation()
             MapControl.SetLocation(this, location);
             MapControl.SetNormalizedAnchorPoint(this, anchor);
         }
+
+        //TODO: implement xamarin view to UWP
+        private async void TransformXamarinViewToUWPBitmap(Pin outerItem, ContentControl nativeItem)
+        {
+            if (outerItem?.Icon?.Type == BitmapDescriptorType.View && outerItem?.Icon?.View != null)
+            {
+                //var iconView = outerItem.Icon.View;
+                //var nativeView = await Utils.ConvertFormsToNative(iconView, new Rectangle(0, 0, (double)Utils.DpToPx((float)iconView.WidthRequest), (double)Utils.DpToPx((float)iconView.HeightRequest)), Platform.Android.Platform.CreateRenderer(iconView));
```

Comment:
This code is never used and can be deleted. I only put it there as a reminder to implement the Xamarin Forms View to Pin on UWP. But it was a lot more complicated than I had anticipated so I abandoned it. It is safe to delete (its copied and pasted from android anyway)